Source/EasyNetQ/Internals/AsyncLock.cs

```diff
@@ -4,7 +4,13 @@
 
 namespace EasyNetQ.Internals
 {
-    public sealed class AsyncLock
+    /// <summary>
+    ///     This is an internal API that supports the EasyNetQ infrastructure and not subject to
+    ///     the same compatibility as public APIs. It may be changed or removed without notice in
+    ///     any release. You should only use it directly in your code with extreme caution and knowing that
+    ///     doing so can result in application failures when updating to a new EasyNetQ release.
+    /// </summary>
+    internal sealed class AsyncLock : IDisposable
```

Comment:
public or internal?

Reply:
It was a typo and it will be fixed in further PRs.